markdown/events/2020/hackathon-july-2020.md

```diff
@@ -57,20 +57,35 @@ To participate, all you need is a computer with an internet connection.
 Presentations will be run using Zoom and live-streamed to YouTube,
 where they will be also be available afterwards. Discussion and collaboration
 will be handled in dedicated channels in the nf-core Slack organisation.
+**YouTube streams are embedded in the page below.**
 
 Relevant links will be emailed to participants ahead of the event. For more
-details or if you have questions, please post a message in the nf-core Slack.
+details or if you have questions, please post a message in the
+[`#hackathon-july-2020` channel](https://nfcore.slack.com/channels/hackathon-july-2020)
+in the nf-core Slack.
 See [https://nf-co.re/join](https://nf-co.re/join) for information on how to join.
 
 All nf-core code is open source with the MIT licence and available on
 GitHub: [https://github.com/nf-core](https://github.com/nf-core)
 
 This event is being organised by the nf-core [core team](https://nf-co.re/about).
 
-### Hackathon project page
+### Hackathon projects
 
+Work within the hackathon will be loosely gathered into projects.
+Each group has its own Slack channel:
+
+* [`#hackathon-july-2020-new-pipelines`](https://nfcore.slack.com/channels/hackathon-july-2020-new-pipelines) - New pipeline development
+* [`#hackathon-july-2020-existing-pipelines`](https://nfcore.slack.com/channels/hackathon-july-2020-existing-pipelines) - Existing nf-core pipeline development
+* [`#hackathon-july-2020-tools`](https://nfcore.slack.com/channels/hackathon-july-2020-tools) - Improving nf-core framework tools (written in Python)
+* [`#hackathon-july-2020-modules-dsl2`](https://nfcore.slack.com/channels/hackathon-july-2020-modules-dsl2) - Nextflow DSL2 modules
+
+Tasks, in the form of GitHub issues, can be gathered and organised within the hackathon project board on GitHub:
 [https://github.com/orgs/nf-core/projects/6](https://github.com/orgs/nf-core/projects/6)
 
+If you'd like to form sub-groups, for example to work on a specific pipeline, that's fine.
```

Comment:
```suggestion
For the wrap-up daily sessions, the progress made during the day will presented in the form of HackMD slides. To show your participation in the Hackathon groups, you can contribute to the slides. Each group has its own slides deck:

* New pipelines slides: [Edit](https://hackmd.io/U3hNik2AQG2bor_4pWlvfQ), [View](https://hackmd.io/@nf-core-hackathon-july-2020/ByaotZZkv)
* Existing pipelines slides: [Edit](https://hackmd.io/sVE8c7XxSW-QyCBaDAwxMg), [View](https://hackmd.io/@nf-core-hackathon-july-2020/Hk7HqW-JD)
* Tools slides: [Edit](https://hackmd.io/Okx7xOUvTGW4NMVropSttg), [View](https://hackmd.io/@nf-core-hackathon-july-2020/r1ZJiZWkP)
* DSL2 modules slides: [Edit](https://hackmd.io/CMmZaDl2QWK3C7ZBDzffPw), [View](https://hackmd.io/@nf-core-hackathon-july-2020/By4n2-bkD)

If you'd like to form sub-groups, for example to work on a specific pipeline, that's fine.
```

Reply:
These are the links to the HackMD slides. Core members can also see the [team board](https://hackmd.io/team/nf-core-hackathon-july-2020?nav=overview) and create new notes, and logged in participants can edit and view the slides links.